app/services/participant_identity_resolver.rb

```diff
@@ -31,7 +31,7 @@ def call
         ParticipantIdentity
          .joins(:npq_participant_profiles, npq_applications: [:npq_course, { npq_lead_provider: :cpd_lead_provider }])
          .where(npq_courses: { identifier: course_identifier })
-         .where(npq_applications: { npq_lead_providers: { cpd_lead_provider: } })
+         .where(npq_applications: { lead_provider_approval_status: "accepted", npq_lead_providers: { cpd_lead_provider: } })
```

Comment:
Just had an afterthought; would this cause the identity resolver to fail for users that don't have an accepted NPQ application? is that a valid state?